app/controllers/pages_controller.rb

```diff
@@ -1,12 +1,38 @@
 class PagesController < ApplicationController
+  before_action :require_website, only: :show
+  before_action :require_page, only: :show
```

Comment:
Since these are only required in one place, can we move them from callbacks to the controller action itself?